core/src/test/scala/org/broadinstitute/dsde/rawls/googleProject/GoogleProjectRegistrationRepositorySpec.scala

```diff
@@ -86,4 +86,59 @@ class GoogleProjectRegistrationRepositorySpec extends AnyFlatSpec with TestDrive
 
   }
 
+  it should "retrieve a google project registration by id" in withDefaultTestDatabase {
+    val repo = new GoogleProjectRegistrationRepository(slickDataSource)
+    val googleProjectReg = makeGoogleProjectRegistration(testData.testProject1)
+
+    Await.result(repo.registerGoogleProject(googleProjectReg), Duration.Inf)
+    val result = Await.result(repo.getGoogleProjectRegistration(googleProjectReg.googleProjectId), Duration.Inf)
+
+    result match {
+      case Some(project) => assertResult(googleProjectReg)(project)
+      case None          => fail(s"Expected Some(${googleProjectReg}) but got None")
+    }
+  }
+
+  it should "return None if google project registration id does not exist" in withDefaultTestDatabase {
+    val repo = new GoogleProjectRegistrationRepository(slickDataSource)
+    val nonExistentId = GoogleProjectId(UUID.randomUUID().toString)
+
+    val result = Await.result(repo.getGoogleProjectRegistration(nonExistentId), Duration.Inf)
+
+    assertResult(None) {
+      result
+    }
+  }
+
+  it should "retrieve multiple google project registrations by ids" in withDefaultTestDatabase {
+    val repo = new GoogleProjectRegistrationRepository(slickDataSource)
+    val googleProjectReg1 = makeGoogleProjectRegistration(testData.testProject1)
+    val googleProjectReg2 = makeGoogleProjectRegistration(testData.testProject2)
+
+    Await.result(repo.registerGoogleProject(googleProjectReg1), Duration.Inf)
+    Await.result(repo.registerGoogleProject(googleProjectReg2), Duration.Inf)
+
+    val result = Await.result(
+      repo.getGoogleProjectRegistrations(Set(googleProjectReg1.googleProjectId, googleProjectReg2.googleProjectId),
+                                         None,
+                                         10,
+                                         0
+      ),
+      Duration.Inf
+    )
+
+    assert(result.contains(googleProjectReg1))
+    assert(result.contains(googleProjectReg2))
```

Comment:
using Matchers:
```
result should contain theSameElementsAs List(googleProjectReg1, googleProjectReg2)
```
in this case, the Matcher DSL is more precise. The existing code would still pass if the result was `List(googleProjectReg1, googleProjectReg2, googleProjectReg3, googleProjectReg4)`. You could also solve this by adding `assertResult(2)(result.size)` and not using Matchers.